src/app/plan/duck/actions.ts

```diff
@@ -1,5 +1,10 @@
 import { createActions } from 'reduxsauce';
 
-const { Creators, Types } = createActions({});
+const { Creators, Types } = createActions({
+  migrationPlanFetchSuccess: ['migrationPlanList'],
```

Comment:
Think this should be `migPlan`, we should generally use `migX` when talking about the actual data objects that are getting posted, it's what we settled on as our prefix for our kube objects. The reason was, `MigrationMigration` is pretty weird, and we've been somewhat branding ourselves as "Mig", so we went with that. Ex: `MigPlan, MigCluster, MigStage, MigMigration`.